incubator/hnc/pkg/config/default_config.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import (
+	api "github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/api/v1alpha1"
+)
+
+// GetDefaultConfig creates the default configuration for HNCConfiguration Spec.
+// By default, HNC configuration should always propagate Roles and RoleBindings.
+// See details in http://bit.ly/hnc-type-configuration
+func GetDefaultConfig() *api.HNCConfigurationSpec {
+	return &api.HNCConfigurationSpec{
```

Comment:
Can you please switch to two-space tabs to align with the rest of the code? Thanks!

Reply:
I have no idea what I was thinking. Ignore this.